Assets/Scripts/Chat/ChatChannel.cs

```diff
@@ -7,4 +7,5 @@ public struct ChatChannel
     public string Name;
     public string Abbreviation;
     public Color Color;
+    public bool Unhidable;

```

Comment:
Negatives can sometimes cause confusing, for example in `if (!Unhidable)`. Prefer `Hidable`.